src/scripts/pg_hba.conf

```diff
@@ -87,18 +87,18 @@
 # maintenance (custom daily cronjobs, replication, and similar tasks).
 #
 # Database administrative login by Unix domain socket
-local   all             postgres                                trust
+local   all             postgres                                peer
 
 # TYPE  DATABASE        USER            ADDRESS                 METHOD
 
 # "local" is for Unix domain socket connections only
 local   all             all                                     peer
 # IPv4 local connections:
 #host    all             all             127.0.0.1/32            scram-sha-256
-host    all             all             127.0.0.1/32            password
+host    all             all             127.0.0.1/32            trust
```

Comment:
We should make sure the port is not exposed to the public. I'm not sure about the networking logic in Docker and what will be the source IP for requests from inside the Docker network/from the Docker host.

Reply:
i think it's ok for localhost. There is also a host 0.0.0.0 there, but it asks for password

Reply:
So, yes. "localhost" within a docker container means the virtual network that's in the docker container. Not even the *actual* localhost of the machine counts for that as docker re-assigns the "real" localhost a different IP address (usually 172.17.42.1) for the container context.

But that is why the standard debian-packaged `pg_hba.conf` *doesn't* do this, because then all accounts on the machine (assuming a classic multi-user Unix server) would be able to log in as any other user within the database.